tests/test-cases/include-local-wildcard/.gitlab-ci-2.yml

```diff
@@ -0,0 +1,3 @@
+---
+include:
+  local: '/**/*.gitlab-ci.yml'
```

Comment:
Have you tested this in gitlab.com ? 

i don't think it's able to parse `/**/*.gitlab-ci.yml`